core/src/test/scala/integration/kafka/admin/ReassignPartitionsIntegrationTest.scala

```diff
@@ -127,7 +127,6 @@ class ReassignPartitionsIntegrationTest extends ZooKeeperTestHarness {
     )
 
     val verifyAssignmentResult = runVerifyAssignment(cluster.adminClient, assignment, false)
-    assertTrue(verifyAssignmentResult.partsOngoing)
     assertFalse(verifyAssignmentResult.movesOngoing)
```

Comment:
I checked and confirmed that this change makes sense, because there might not have on-going reassignment at this moment if the reassignment completed before we verify it. 

However, in this case, I don't think we verify the `moveOngoing` makes sense here, either, since it might passed just because the reassignment completed before the verification. I think we can directly remove all these 3 lines and directly wait for the reassignment completion. What do you think?

Reply:
verifying `moveOngoing` here makes sense because we don't perform intra-broker replica assignments(move log dirs) in these tests, so `movesOngoing` should always be false.